conda_build/metadata.py

```diff
@@ -1853,6 +1854,10 @@ def info_index(self):
             build_noarch = self.get_value("build/noarch")
             if build_noarch:
                 d["noarch"] = build_noarch
+        elif self.python_version_independent:
+            # This is a hack to make conda/mamba/micromamba compile the pure python files
+            # and for micromamba to move the files in site-packages to the correct dir.
```

Comment:
Which versions require this hack? All of them?

Reply:
I added a clarification to the comment here.